api/blocks/blocks.go

```diff
@@ -47,6 +54,16 @@ func (b *Blocks) handleGetBlock(w http.ResponseWriter, req *http.Request) error
 		return err
 	}
 
+	if raw == "true" {
```

Comment:
Should also consider that the priority of `raw` is higher than `expanded`.

Reply:
`raw` is already taking precedence over `expanded`, should it be the other way around?
 https://github.com/vechain/thor/blob/1f79185698a38845980de1a8c2a8b0ff99f143d2/api/blocks/blocks.go#L57

Reply:
My bad, you're right